index.js

```diff
@@ -22,4 +25,30 @@ export class Core {
       this.logs
     )
   }
+
+  async setup () {
```

Comment:
Should we make this method private?

```suggestion
  async #setup () {
```